### DISCUSSION TITLE
Organize collection?

### DISCUSSION THREAD

**Discussion by eelstork** [starikcetin/awesome-upm#7] [Meta]
There's a few things I think could be tried to make this more useful but in short, I'm wondering if we could just grab the package name and description based on a URL to the JSON.
Also, if we setup a few cats, would help finding interesting things (I think the category could be added to the JSON without breaking anything)

Some of this might overlap with OpenUPM... maybe.

**Comment by starikcetin** [comment]
That would be possible, but it requires a frontend as well, which Unity doesn't have, and it would be a huge project for my current schedule.

So yeah, we better wait for OpenUPM for that purpose.

For categories, I think we should wait for a little more for more packages to accumulate. But if you are up to it, feel free.

**Comment by starikcetin** [comment]
Working on this: https://github.com/starikcetin/awesome-upm/issues/23
